tests/microos/cockpit_check.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic check for cockpit service
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+use utils qw(systemctl);
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Install cockpit if needed, this is needed for DVD flavor where
+    # Cockpit pattern is not selected during install
+    if (script_run('rpm -q cockpit') != 0) {
```

Comment:
maybe we need to have here something like `if(get_var('FALVOR', ...))` instead ? so when package will be missing in unexpected flavor we will fail here ?

Reply:
But there are 2 options in DVD installation flavor:
1) Add pattern during installation (which we don't do)
2) Install it later

So, it doesn't depend on the flavor, it depends on the install setup. I think it's a good exercise to install it, as we have it pre-installed in the Image flavor.

Reply:
> But there are 2 options in DVD installation flavor:
> 
>     1. Add pattern during installation (which we don't do)
> 
>     2. Install it later
> 
> 
> So, it doesn't depend on the flavor, it depends on the install setup. I think it's a good exercise to install it, as we have it pre-installed in the Image flavor.

I meant something different , in case of DVD it is expected that you may install it , but in other cases except DVD it suppose to be installed beforehand. And if with whatever reason when you boot with raw image package will be not there current flow will silently install it . I would treat this as missed bug and would expect that raw image test will fail in such a case

Reply:
Good point. Noted in the poo ticket. Thanks!

Reply:
I just found out that openSUSE MicroOS Image doesn't have it installed by default... I am not sure if we should make that check, cause there are different decisions for SLE and openSUSE that might change in the future...